.github/workflows/check-e2e.yml

```diff
@@ -58,6 +61,18 @@ jobs:
         if: steps.playwright-dep-cache.outputs.cache-hit == 'true'
         run: npx playwright install-deps
 
+      - id: auth
+        uses: google-github-actions/auth@v1
+        with:
+          workload_identity_provider: 'projects/281042598092/locations/global/workloadIdentityPools/github-actions/providers/google-cloud'
+          service_account: 'github-actions@radicle-services.iam.gserviceaccount.com'
+
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@v1
+
+      - name: Download radicle-httpd executable
+        run: gsutil cp gs://heartwood-artifacts/119870d59683c446b2377974ab31bc1fe1b98d43/radicle-httpd ./
+
```

Comment:
This `119870d59683c446b2377974ab31bc1fe1b98d43` is a commit sha within the `heartwood` repo.  As soon as <https://github.com/radicle-dev/heartwood/pull/196> gets merged, any SHA from `master` is going to be allowed.